addon-test-support/@ember/test-helpers/-utils.js

```diff
@@ -1,6 +1,7 @@
 import { Promise } from 'rsvp';
 
-export const nextTick = setTimeout;
+export const nextTick =
+  typeof Promise === 'undefined' ? setTimeout : cb => Promise.resolve().then(cb);
```

Comment:
doesn't that conflict with the `nextTickPromise()` description below which talks about "the next turn of the event loop"?